lib/StatusIndicator/styles.scss

```diff
@@ -17,8 +17,11 @@ $status-indicator-bordered-radius: .25rem !default;
   white-space: nowrap;
 
   .status-indicator__children {
-    color: $color-brand-dark;
-    font-size: $typo-size-small;
+    color: $color-text-light;
+    font-size: $typo-size-slight;
+    margin-left: $status-indicator-margin-left;
+    font-weight: 500;
+    margin-top: 8px;
```

Comment:
Maybe we could set this based on the base layout spacing setting? e.g. `$layout-spacing-base/2` = 10px